ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/TierSwitcher.kt

```diff
@@ -82,7 +83,7 @@ internal fun TierSwitcher(
 ) {
     val selectedIndex = tiers.indexOf(selectedTier)
     var totalWidthPx by remember { mutableStateOf(0) }
-    var totalHeightPx by remember { mutableStateOf(0) }
+    var totalHeightRowDp by remember { mutableStateOf(40.dp) }
```

Comment:
Any reason for this to start at 40.dp?

Reply:
@tonidero Oh, for snapshot tests 😅 Otherwise the highlighted background selector box thing doesn't show because its height 0 🤷‍♂️  It's _nearly_ instant when running by a real person snapshots weren't working

Reply:
Yeah snapshot tests don't seem to take the snapshot correctly when there are post layout measurements. Makes sense then!

Reply:
I maybe should have added that as a comment 😅 I'll do that in the `multi-tier-paywall` branch before I merge it into `main` 💪